### PR TITLE
updated styling

### PR DESCRIPTION
**Pull request by JamesDunton**

### CODE REVIEW

**Review by conninator2000 — APPROVED** [review]
Looks good

**Review by hunter-bennett28 — APPROVED** [review]

**Review by hunter-bennett28 — CHANGES_REQUESTED** [review]
SubtaskMemberSelectionComponent state object needs some work, essentially unused and random values that are never set or used. On selecting a member to assign, db is updated by the UI is not so reopening it shows no assigned member. 

UI also needs fixing to not have overlapping content

File: src/components/MemberComponent.js [hunk 1]
```diff
@@ -13,14 +13,65 @@ import {
   IconButton,
```
Comment (conninator2000) [inline src/components/MemberComponent.js:13]: I added styling to this to have the modal reflect the product backlog

File: src/components/Retrospective/SprintRetrospectiveComponent.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ const SprintRetrospectiveComponent = ({ loggedIn, displayPopup }) => {
                   updatedSubtask.hoursWorked =
                     parseInt(inputData.hoursWorked) + parseInt(updatedSubtask.hoursWorked);
 
-                  updatedSubtask.hoursEstimated = parseInt(updatedSubtask.hoursEstimated);
+                  updatedSubtask.hoursEstimated = parseInt(inputData.hoursEstimated);
```
Comment (conninator2000) [inline src/components/Retrospective/SprintRetrospectiveComponent.js:119]: Originally returned the same data it was trying to set so no update occurred

**Review by hunter-bennett28 — APPROVED** [review]

**Pull request merged**